source/module_cell/klist.cpp

```diff
@@ -654,19 +591,19 @@ void K_Vectors::ibz_kpoint(
     ModuleBase::Matrix3 inv(-1, 0, 0, 0, -1, 0, 0, 0, -1);
     ModuleBase::Matrix3 ind(1, 0, 0, 0, 1, 0, 0, 0, 1);
 
-    int nrotkm=0;
-    if(use_symm)
+    int nrotkm = 0;
+    if (use_symm)
     {
         // bravais type of reciprocal lattice and k-lattice
         double b_const[6];
         double b0_const[6];
         double bk_const[6];
         double bk0_const[6];
-        int bbrav=15;
-        int bkbrav=15;
+        int bbrav = 15;
+        int bkbrav = 15;
```

Comment:
As I understand it, `bbrav` and `bkbrav` are used to label the type of Bravais lattice, and the initial value is set to `15` because there are only `14` Bravais lattices in 3D space, and this value is equivalent to the default value.

As for `bbrav`, the first letter `b` means `the reciprocal space`. This may be a personal habit of the original author, I find he likes to use `a` for real space and `b` for reciprocal space. Then `brav` means `Bravais lattice`, and `k` means `k-lattice`.

Maybe I can change them into `bbrav_type` and `bkbrav_type`?